lib/commands/debug.ts

```diff
@@ -21,6 +21,8 @@
 				return (() => {
 					let projectData: IProjectData = this.$injector.resolve("projectData");
 
+					this.$logger.info(`Successfully synced application ${deviceAppData.appIdentifier} on device ${deviceAppData.device.deviceInfo.identifier}.`);
```

Comment:
This message is repeated in the platform-livesync-service-base, so I vote for moving both messages into the livesync-service. This way the chance to change/delete it would be lower. 
Also it would be nice to add a comment why this message is so important